src/webgl/GeometryBuilder.js

```diff
@@ -46,13 +46,9 @@ class GeometryBuilder {
    * transformations.
    */
   addGeometry(input) {
-    this.renderer.uMVMatrix =
-     this.renderer.uModelMatrix.mult(this.renderer.uViewMatrix);
-    this.hasTransform = !this.renderer.uMVMatrix.mat4
```

Comment:
I think we still want to set `this.hasTransform`, but just using the model matrix only rather than `uMVMatrix`.